commit 7f85f7b70d947c08f4424bf614b5d66bb6d9a0ac
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Jul 22 19:09:38 2021 -0300

    Correções de matriz de confusão -- Aline
    
    nmodXcompound
    nummodXflat:name --> fazer por regra
    amodXadvcl
    amodXobl

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 9efd8aa..dbc5e7e 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -280,9 +280,9 @@
 # text = Figura 1: Localização da Bacia de Pelotas
 # sent_id = 247-20140910-MONOGRAFIA_0-7
 1	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	0	root	O	_
-2	1	1	PROPN	_	Number=Sing	1	flat:name	O	_
+2	1	1	NUM	_	Number=Sing	1	nummod	O	_
 3	:	:	PUNCT	_	_	4	punct	O	_
-4	Localização	Localização	PROPN	_	Gender=Fem|Number=Sing	1	appos	O	_
+4	Localização	Localização	PROPN	_	Gender=Fem|Number=Sing	1	parataxis	O	_
 5-6	da	_	_	_	_	_	_	_	_
 5	de	de	ADP	_	_	4	flat:name	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	flat:name	O	_
@@ -2456,7 +2456,7 @@
 8	em	em	ADP	_	_	10	case	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 10	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	7	obl	O	_
-11	4	4	PROPN	_	Number=Sing	10	flat:name	O	_
+11	4	4	NUM	_	Number=Sing	10	nummod	O	_
 12	,	,	PUNCT	_	_	7	punct	O	_
 13	com	com	ADP	_	_	14	case	O	_
 14	integração	integração	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
@@ -3150,7 +3150,7 @@
 24	em	em	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
 26	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	23	obl	O	_
-27	7..	7..	PROPN	_	Number=Sing	26	flat:name	O	_
+27	7..	7..	NUM	_	Number=Sing	26	nummod	O	tokenização
 
 # text = Ordovício-Siluriana (Supersequência Rio Ivaí): representada pelo Grupo Rio Ivaí, possui as formações Alto Garças (conglomerados e arenitos conglomeráticos), Iapó (diamictitos) e Vila Maria (folhelhos e siltitos)..
 # sent_id = 247-20140910-MONOGRAFIA_0-87
@@ -3768,7 +3768,7 @@
 46	brasileiras	brasileiro	ADJ	_	Gender=Fem|Number=Plur	45	amod	O	_
 47	(	(	PUNCT	_	_	48	punct	O	_
 48	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	45	appos	O	_
-49	8	8	PROPN	_	Number=Sing	48	flat:name	O	_
+49	8	8	NUM	_	Number=Sing	48	nummod	O	_
 50	)	)	PUNCT	_	_	48	punct	O	_
 51	.	.	PUNCT	_	_	7	punct	O	_
 52	.	.	PUNCT	_	_	7	punct	O	_
@@ -10115,18 +10115,18 @@
 50	mostra	mostrar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	35	conj	O	_
 51	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	50	expl	O	_
 52	assimétrico	assimétrico	ADJ	_	Gender=Masc|Number=Sing	50	xcomp	O	_
-53	em	em	ADP	_	_	59	case	O	_
+53	em	em	ADP	_	_	57	case	O	_
 54	relação	relação	NOUN	_	Gender=Fem|Number=Sing	53	fixed	O	_
 55-56	ao	_	_	_	_	_	_	_	_
 55	a	a	ADP	_	_	53	fixed	O	_
-56	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	59	det	O	_
-57	alto	alto	ADJ	_	Gender=Masc|Number=Sing	59	amod	O	_
-58	principal	principal	ADJ	_	Gender=Masc|Number=Sing	59	amod	O	_
-59	identificado	identificar	NOUN	_	Gender=Masc|Number=Sing	50	obl	O	_
+56	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	57	det	O	_
+57	alto	alto	NOUN	_	Gender=Masc|Number=Sing	50	obl	O	_
+58	principal	principal	ADJ	_	Gender=Masc|Number=Sing	57	amod	O	_
+59	identificado	identificar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	57	acl	O	_
 60-61	no	_	_	_	_	_	_	_	_
 60	em	em	ADP	_	_	62	case	O	_
 61	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	62	det	O	_
-62	embasamento	embasamento	NOUN	_	Gender=Masc|Number=Sing	59	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
+62	embasamento	embasamento	NOUN	_	Gender=Masc|Number=Sing	59	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
 63	.	.	PUNCT	_	_	11	punct	O	_
 64	.	.	PUNCT	_	_	11	punct	O	_
 
@@ -19205,7 +19205,7 @@
 
 # text = CAPÍTULO 7
 # sent_id = 241-20140227-MONOGRAFIA_0-255
-1	CAPÍTULO	capítulo	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+1	CAPÍTULO	capítulo	NOUN	_	Gender=Masc|Number=Sing	0	root	O	eliminar
 2	7	7	PROPN	_	Number=Sing	1	flat:name	O	_
 3	REFERÊNCIAS	refervncia	NOUN	_	Gender=Fem|Number=Plur	1	flat:name	O	_
 
@@ -24990,7 +24990,7 @@
 6	conteúdo	conteúdo	NOUN	_	Gender=Masc|Number=Sing	4	conj	O	_
 7	fóssil	fóssil	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
 8	predominantemente	predominantemente	ADV	_	_	9	advmod	O	_
-9	lacustres	lacustre	ADJ	_	Gender=Masc|Number=Plur	6	amod	B=LOCAL:GEO	_
+9	lacustres	lacustre	ADJ	_	Gender=Masc|Number=Plur	4	amod	B=LOCAL:GEO	_
 10-11	na	_	_	_	_	_	_	_	_
 10	em	em	ADP	_	_	12	case	O	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
@@ -28319,7 +28319,7 @@
 28	intervalo	intervalo	NOUN	_	Gender=Masc|Number=Sing	25	obl	O	_
 29	com	com	ADP	_	_	31	case	O	_
 30	delgadas	delgado	ADJ	_	Gender=Fem|Number=Plur	31	amod	O	_
-31	camadas	camada	NOUN	_	Gender=Fem|Number=Plur	25	obl	O	_
+31	camadas	camada	NOUN	_	Gender=Fem|Number=Plur	28	nmod	O	_
 32	(	(	PUNCT	_	_	34	punct	O	_
 33	4	4	NUM	_	NumType=Card	34	nummod	O	_
 34	camadas	camada	NOUN	_	Gender=Fem|Number=Plur	31	appos	O	_
@@ -29668,7 +29668,7 @@
 27	confere	conferir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	14	conj	O	_
 28	a	a	ADP	_	_	31	case	O	_
 29	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	31	det	O	_
-30	nível/	nível/	ADJ	_	Gender=Masc|Number=Sing	31	amod	O	_
+30	nível/	nível/	ADJ	_	Gender=Masc|Number=Sing	31	amod	O	tokenização
 31	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	27	iobj	O	_
 32	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	33	det	O	_
 33	caráter	caráter	NOUN	_	Gender=Masc|Number=Sing	27	obj	O	_
@@ -31932,7 +31932,7 @@
 32	por	por	ADP	_	_	35	case	O	_
 33	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	35	det	O	_
 34	delgado	delgado	ADJ	_	Gender=Masc|Number=Sing	35	amod	O	_
-35	intervalo	intervalo	NOUN	_	Gender=Masc|Number=Sing	27	obl	O	_
+35	intervalo	intervalo	NOUN	_	Gender=Masc|Number=Sing	29	conj	O	_
 36	(	(	PUNCT	_	_	38	punct	O	_
 37	1-2	1-2	NUM	_	NumType=Card	38	nummod	O	_
 38	m	m	NOUN	_	Gender=Masc|Number=Plur	35	appos	O	_
@@ -50881,7 +50881,7 @@
 4	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	5	det	O	_
 5	folhas	folha	NOUN	_	Gender=Fem|Number=Plur	2	nmod	O	_
 6	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	7	cop	O	_
-7	empilhadas	empilhado	ADJ	_	Gender=Fem|Number=Plur	8	advcl	O	_
+7	empilhadas	empilhado	ADJ	_	Gender=Fem|Number=Plur	2	acl	O	_
 8	difere	diferir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 9	para	para	ADP	_	_	12	case	O	_
 10	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	12	det	O	_
@@ -55494,7 +55494,7 @@
 5	em	em	ADP	_	_	7	case	O	_
 6	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
 7	espectrofotômetro	espectrofotômetro	NOUN	_	Gender=Masc|Number=Sing	4	obl	O	_
-8	modelo	modelo	ADJ	_	Gender=Masc|Number=Sing	7	amod	O	_
+8	modelo	modelo	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
 9	46	46	NUM	_	NumType=Card	10	nummod	O	_
 10	M350	M350	PROPN	_	Gender=Masc|Number=Sing	7	nmod	O	_
 11	Double	Double	PROPN	_	Number=Sing	10	flat:name	O	_
@@ -59739,9 +59739,9 @@
 47-48	na	_	_	_	_	_	_	_	_
 47	em	em	ADP	_	_	50	case	O	_
 48	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	50	det	O	_
-49	intercamada	intercamado	ADJ	_	Gender=Fem|Number=Sing	50	amod	O	_
-50	planar	planar	NOUN	_	Gender=Fem|Number=Sing	44	obl	O	_
-51	basal	basal	ADJ	_	Gender=Fem|Number=Sing	50	amod	O	_
+49	intercamada	intercamado	NOUN	_	Gender=Fem|Number=Sing	44	obl	O	_
+50	planar	planar	ADJ	_	Gender=Fem|Number=Sing	49	amod	O	_
+51	basal	basal	ADJ	_	Gender=Fem|Number=Sing	49	amod	O	_
 52	.	.	PUNCT	_	_	4	punct	O	_
 53	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -73555,10 +73555,10 @@
 4	normalmente	normalmente	ADV	_	_	5	advmod	O	_
 5	sobrepostas	sobrepor	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	2	xcomp	O	_
 6-7	aos	_	_	_	_	_	_	_	_
-6	a	a	ADP	_	_	9	case	O	_
-7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	9	det	O	_
-8	arenitos	arenito	ADJ	_	Gender=Masc|Number=Plur	9	amod	B=LITOLOGIA	_
-9	Sh	Sh	PROPN	_	Gender=Masc|Number=Sing	5	obl	O	_
+6	a	a	ADP	_	_	8	case	O	_
+7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
+8	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	5	obl	B=LITOLOGIA	_
+9	Sh	Sh	PROPN	_	Gender=Masc|Number=Sing	8	nmod	O	_
 10	,	,	PUNCT	_	_	14	punct	O	_
 11	em	em	ADP	_	_	13	case	O	_
 12	alguns	algum	DET	_	Gender=Masc|Number=Plur|PronType=Ind	13	det	O	_
@@ -113399,7 +113399,7 @@
 38-39	à	_	_	_	_	_	_	_	_
 38	a	a	ADP	_	_	40	case	O	_
 39	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	40	det	O	_
-40	atual	atual	ADJ	_	Gender=Fem|Number=Sing	36	amod	O	_
+40	atual	atual	ADJ	_	Gender=Fem|Number=Sing	37	obl	O	_
 41	,	,	PUNCT	_	_	34	punct	O	_
 42	de	de	ADP	_	_	43	case	O	_
 43	processamento	processamento	NOUN	_	Gender=Masc|Number=Sing	36	nmod	O	_
@@ -158047,7 +158047,7 @@
 10-11	nos	_	_	_	_	_	_	_	_
 10	em	em	ADP	_	_	12	case	O	_
 11	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	12	det	O	_
-12	mesmos	mesmo	ADJ	_	Gender=Masc|Number=Plur	9	amod	O	_
+12	mesmos	mesmo	NOUN	_	Gender=Masc|Number=Plur	8	obl	O	_
 13	(	(	PUNCT	_	_	14	punct	O	_
 14	evidenciados	evidenciar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	9	acl	O	_
 15-16	nos	_	_	_	_	_	_	_	_
@@ -169269,7 +169269,7 @@
 5	de	de	ADP	_	_	7	case	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
 7	escoamento	escoamento	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
-8	anular	anular	VERB	_	Number=Sing|Person=3|VerbForm=Inf	10	advcl	O	_
+8	anular	anular	ADJ	_	Number=Sing|Person=3|VerbForm=Inf	7	amod	O	_
 9	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	10	aux:pass	O	_
 10	considerada	considerar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 11-12	pelo	_	_	_	_	_	_	_	_
@@ -170977,7 +170977,7 @@
 11	expostas	expor	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	2	acl	O	_
 12	a	a	ADP	_	_	13	case	O	_
 13	sistemas	sistema	NOUN	_	Gender=Masc|Number=Plur	11	obj	O	_
-14	óleo/água	óleo/águo	ADJ	_	Gender=Masc|Number=Plur	13	amod	O	_
+14	óleo/água	óleo/águo	NOUN	_	Gender=Masc|Number=Plur	13	nmod	O	_
 15	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	16	aux:pass	O	_
 16	baseada	basear	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 17	em	em	ADP	_	_	18	case	O	_
@@ -178428,7 +178428,7 @@
 10	limpo	limpo	ADJ	_	Gender=Masc|Number=Sing	8	xcomp	O	_
 11	,	,	PUNCT	_	_	8	punct	O	_
 12	caso	caso	SCONJ	_	_	14	mark	O	_
-13	contrário	contrário	ADJ	_	Gender=Masc|Number=Sing	14	advcl	O	_
+13	contrário	contrário	ADJ	_	Gender=Masc|Number=Sing	12	fixed	O	_
 14	seria	ser	VERB	_	Mood=Cnd|Number=Sing|Person=3|VerbForm=Fin	8	advcl	O	_
 15	de	de	SCONJ	_	_	17	mark	O	_
 16	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	17	nsubj	O	_
@@ -183403,11 +183403,11 @@
 7	de	de	ADP	_	_	8	case	O	_
 8	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	6	nmod	B=FLUIDO	_
 9	vegetal	vegetal	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
-10	em	em	ADP	_	_	13	case	O	_
-11	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-12	motor	motor	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
-13	ciclo	ciclo	NOUN	_	Gender=Masc|Number=Sing	4	obl	O	_
-14	diesel	diesel	ADJ	_	Gender=Masc|Number=Sing	13	amod	B=PRODUTO	_
+10	em	em	ADP	_	_	12	case	O	_
+11	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
+12	motor	motor	NOUN	_	Gender=Masc|Number=Sing	4	obl	O	_
+13	ciclo	ciclo	NOUN	_	Gender=Masc|Number=Sing	12	nmod	O	_
+14	diesel	diesel	NOUN	_	Gender=Masc|Number=Sing	12	nmod	B=PRODUTO	_
 15	de	de	ADP	_	_	17	case	O	_
 16	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	17	det	O	_
 17	trator	trator	NOUN	_	Gender=Masc|Number=Sing	13	nmod	B=TRANSPORTE	_
@@ -198339,7 +198339,7 @@
 10	para	para	ADP	_	_	12	case	O	_
 11	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	12	det	O	_
 12	situações	situação	NOUN	_	Gender=Fem|Number=Plur	4	obl	O	_
-13	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	12	compound	O	_
+13	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	12	nmod	O	_
 14	.	.	PUNCT	_	_	4	punct	O	_
 15	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -203915,7 +203915,7 @@
 # sent_id = 271-20150810-MONOGRAFIA_0-448
 1	4.3.6.1	4.3.6.1	NUM	_	NumType=Card	2	nummod	O	_
 2	Usina	Usina	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
-3	piloto	piloto	NOUN	_	Gender=Masc|Number=Sing	2	compound	O	_
+3	piloto	piloto	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 4	de	de	ADP	_	_	5	case	O	_
 5	Biodiesel	Biodiesel	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=PRODUTO	_
 
@@ -205470,7 +205470,7 @@
 9	para	para	ADP	_	_	11	case	O	_
 10	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	situação	situação	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
-12	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	11	compound	O	_
+12	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	11	nmod	O	_
 13	.	.	PUNCT	_	_	4	punct	O	_
 14	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -209097,8 +209097,8 @@
 9	,	,	PUNCT	_	_	4	punct	O	_
 10	tanto	tanto	ADV	_	_	4	advmod	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-12	radical	radical	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
-13	ácido	ácido	NOUN	_	Gender=Masc|Number=Sing	4	obj	O	_
+12	radical	radical	NOUN	_	Gender=Masc|Number=Sing	4	obj	O	_
+13	ácido	ácido	ADJ	_	Gender=Masc|Number=Sing	12	amod	O	_
 14	como	como	ADP	_	_	16	case	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
 16	radical	radical	NOUN	_	Gender=Masc|Number=Sing	4	xcomp	O	_
@@ -214887,8 +214887,8 @@
 1	6.1	6.1	NUM	_	NumType=Card	3	parataxis	O	_
 2	.	.	PUNCT	_	_	3	punct	O	_
 3	Curva	curva	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
-4	de	de	ADP	_	_	3	compound	O	_
-5	calibração	calibração	NOUN	_	Gender=Fem|Number=Sing	3	compound	O	_
+4	de	de	ADP	_	_	5	case	O	_
+5	calibração	calibração	NOUN	_	Gender=Fem|Number=Sing	3	nmod	O	_
 
 # text = Com a leitura da absorbância feita por meio do espectro, dos diferentes padrões preparados com concentrações pré-definidas, plotou-se a curva de calibração da Absorbância versus Concentração de diesel no clorofórmio para o comprimento de ondas (λ) de 260 nm-1, conforme Figura 6..
 # sent_id = 391-20160914-MONOGRAFIA_0-223
@@ -219682,7 +219682,7 @@
 26	espectro	espectro	NOUN	_	Gender=Masc|Number=Sing	22	obj	O	_
 27	desde	desde	ADP	_	_	29	case	O	_
 28	altamente	altamente	ADV	_	_	29	advmod	O	_
-29	hidrofóbicas	hidrofóbico	ADJ	_	Gender=Fem|Number=Plur	26	amod	O	_
+29	hidrofóbicas	hidrofóbico	ADJ	_	Gender=Fem|Number=Plur	22	obl	O	_
 30	a	a	ADP	_	_	32	case	O	_
 31	altamente	altamente	ADV	_	_	32	advmod	O	_
 32	hidrofílicas	hidrofílico	ADJ	_	Gender=Fem|Number=Plur	29	obl	O	_
@@ -229146,10 +229146,10 @@
 35	[	[	PUNCT	_	_	36	punct	O	_
 36	9	9	NUM	_	NumType=Card	34	nmod:appos	O	_
 37	]	]	PUNCT	_	_	36	punct	O	_
-38	para	para	ADP	_	_	41	case	O	_
-39	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	41	det	O	_
-40	zeólita	zeólito	ADJ	_	Gender=Fem|Number=Sing	41	amod	B=LITOLOGIA	_
-41	suporte	suporte	NOUN	_	Gender=Fem|Number=Sing	32	obl	O	_
+38	para	para	ADP	_	_	40	case	O	_
+39	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	40	det	O	_
+40	zeólita	zeólito	NOUN	_	Gender=Fem|Number=Sing	32	obl	B=LITOLOGIA	_
+41	suporte	suporte	NOUN	_	Gender=Fem|Number=Sing	40	nmod	O	_
 42	com	com	ADP	_	_	44	case	O	_
 43	menor	pequeno	ADJ	_	Gender=Masc|Number=Sing	44	amod	O	_
 44	SAR	SAR	PROPN	_	Gender=Masc|Number=Sing	41	nmod	O	_
@@ -237236,9 +237236,9 @@
 16	por	por	ADP	_	_	17	case	O	_
 17	Heilbron	Heilbron	PROPN	_	Gender=Masc|Number=Sing	15	obl:agent	O	_
 18	et	et	PROPN	_	Number=Sing	17	flat:name	O	_
-19	al.(	al.(	X	_	_	17	flat:name	O	_
-20	2000	2000	NUM	_	NumType=Card	17	flat:name	O	_
-21	)	)	PUNCT	_	_	17	punct	O	_
+19	al.(	al.(	X	_	_	17	flat:name	O	tokenização
+20	2000	2000	NUM	_	NumType=Card	17	nmod:appos	O	_
+21	)	)	PUNCT	_	_	20	punct	O	_
 22	de	de	ADP	_	_	23	case	O	_
 23	Ocidental	Ocidental	PROPN	_	Gender=Fem|Number=Sing	17	nmod	O	_
 24	,	,	PUNCT	_	_	25	punct	O	_
@@ -241513,9 +241513,9 @@
 28	João	João	PROPN	_	Number=Sing	24	flat:name	O	_
 29	)	)	PUNCT	_	_	21	punct	O	_
 30	apresentaram	apresentar	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	0	root	O	_
-31	em	em	ADP	_	_	34	case	O	_
-32	seus	seu	DET	_	Gender=Masc|Number=Plur|PronType=Prs	34	det	O	_
-33	depocentros	depocentro	ADJ	_	Gender=Masc|Number=Plur	34	amod	O	_
+31	em	em	ADP	_	_	33	case	O	_
+32	seus	seu	DET	_	Gender=Masc|Number=Plur|PronType=Prs	33	det	O	_
+33	depocentros	depocentro	NOUN	_	Gender=Masc|Number=Plur	30	obl	O	_
 34	profundidades	profundidade	NOUN	_	Gender=Fem|Number=Plur	30	obj	O	_
 35	máximas	grande	ADJ	_	Gender=Fem|Number=Plur	34	amod	O	_
 36	em	em	ADP	_	_	40	case	O	_
@@ -247057,12 +247057,12 @@
 33	deu	dar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	30	acl:relcl	O	_
 34	origem	origem	NOUN	_	Gender=Fem|Number=Sing	33	obj	O	_
 35-36	à	_	_	_	_	_	_	_	_
-35	a	a	ADP	_	_	38	case	O	_
-36	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	38	det	O	_
-37	suíte	suíte	ADJ	_	Gender=Fem|Number=Sing	38	amod	O	_
-38	alcalina	alcalina	NOUN	_	Gender=Fem|Number=Sing	33	obl	O	_
+35	a	a	ADP	_	_	37	case	O	_
+36	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	37	det	O	_
+37	suíte	suíte	NOUN	_	Gender=Fem|Number=Sing	33	obl	O	_
+38	alcalina	alcalina	ADJ	_	Gender=Fem|Number=Sing	37	amod	O	_
 39	,	,	PUNCT	_	_	40	punct	O	_
-40	acompanhado	acompanhar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	38	acl	O	_
+40	acompanhado	acompanhar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	24	acl	O	_
 41	a	a	ADP	_	_	42	case	O	_
 42	leste	leste	NOUN	_	Gender=Masc|Number=Sing	40	obl	O	_
 43	por	por	ADP	_	_	46	case	O	_
@@ -247609,7 +247609,7 @@
 9-10	da	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
-11	física	físico	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
+11	física	físico	NOUN	_	Gender=Fem|Number=Sing	8	obl	O	_
 12	matemática	matemática	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
 13	(	(	PUNCT	_	_	15	punct	O	_
 14	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	15	det	O	_
@@ -257425,10 +257425,10 @@
 56	)	)	PUNCT	_	_	46	punct	O	_
 57	referentes	referente	ADJ	_	Gender=Fem|Number=Plur	46	amod	O	_
 58-59	às	_	_	_	_	_	_	_	_
-58	a	a	ADP	_	_	61	case	O	_
-59	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	61	det	O	_
-60	suítes	suíte	ADJ	_	Gender=Fem|Number=Plur	61	amod	O	_
-61	alcalinas	alcalino	NOUN	_	Gender=Fem|Number=Plur	57	obl	B=QUÍMICA	_
+58	a	a	ADP	_	_	60	case	O	_
+59	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	60	det	O	_
+60	suítes	suíte	NOUN	_	Gender=Fem|Number=Plur	57	obl	O	_
+61	alcalinas	alcalino	ADJ	_	Gender=Fem|Number=Plur	60	amod	B=QUÍMICA	_
 62	.	.	PUNCT	_	_	8	punct	O	_
 63	.	.	PUNCT	_	_	8	punct	O	_
 
@@ -264143,8 +264143,8 @@
 37-38	do	_	_	_	_	_	_	_	_
 37	de	de	ADP	_	_	40	case	O	_
 38	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	40	det	O	_
-39	complexo	complexo	ADJ	_	Gender=Masc|Number=Sing	40	amod	O	_
-40	alcalino	alcalino	NOUN	_	Gender=Masc|Number=Sing	36	obl	B=QUÍMICA	_
+39	complexo	complexo	NOUN	_	Gender=Masc|Number=Sing	31	nmod	O	_
+40	alcalino	alcalino	ADJ	_	Gender=Masc|Number=Sing	39	amod	B=QUÍMICA	_
 41	,	,	PUNCT	_	_	28	punct	O	_
 42	dá	dar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	22	ccomp	O	_
 43	origem	origem	NOUN	_	Gender=Fem|Number=Sing	42	obj	O	_
@@ -275652,8 +275652,8 @@
 6-7	à	_	_	_	_	_	_	_	_
 6	a	a	ADP	_	_	9	case	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
-8	suíte	suíte	ADJ	_	Gender=Fem|Number=Sing	9	amod	O	_
-9	alcalina	alcalina	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
+8	suíte	suíte	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
+9	alcalina	alcalina	ADJ	_	Gender=Fem|Number=Sing	8	amod	O	_
 10	de	de	ADP	_	_	11	case	O	_
 11	Arraial	Arraial	PROPN	_	Gender=Masc|Number=Sing	9	nmod	O	_
 12-13	do	_	_	_	_	_	_	_	_
@@ -288801,7 +288801,7 @@
 2	.	.	PUNCT	_	_	3	punct	O	_
 3	Fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	0	root	B=FLUIDO	_
 4	de	de	ADP	_	_	5	case	I=FLUIDO	_
-5	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	3	compound	B=LITOLOGIA|I=FLUIDO	_
+5	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	3	nmod	B=LITOLOGIA|I=FLUIDO	_
 
 # text = Fluidos de perfuração são misturas complexas de sólidos, líquidos, produtos químicos e, por vezes, até gases..
 # sent_id = 119-20141209-TESEMSC_0-81
@@ -316712,8 +316712,8 @@
 8	de	de	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
 10	espaço	espaço	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
-11	anular	anular	VERB	_	Number=Sing|VerbForm=Inf	4	advcl	O	_
-12	formado	formar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	11	xcomp	O	_
+11	anular	anular	ADJ	_	Number=Sing|VerbForm=Inf	10	amod	O	_
+12	formado	formar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	10	acl	O	_
 13-14	pelas	_	_	_	_	_	_	_	_
 13	por	por	ADP	_	_	15	case	O	_
 14	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	15	det	O	_
@@ -316936,12 +316936,12 @@
 8	por	por	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
 10	espaço	espaço	NOUN	_	Gender=Masc|Number=Sing	5	obl	O	_
-11	anular	anular	VERB	_	Number=Sing|VerbForm=Inf	5	advcl	O	_
+11	anular	anular	ADJ	_	Number=Sing|VerbForm=Inf	11	amod	O	_
 12	,	,	PUNCT	_	_	11	punct	O	_
 13-14	do	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
-15	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	11	obl	B=FLUIDO	_
+15	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	7	nmod	B=FLUIDO	_
 16	de	de	ADP	_	_	17	case	I=FLUIDO	_
 17	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	15	nmod	B=LITOLOGIA|I=FLUIDO	_
 18	,	,	PUNCT	_	_	19	punct	O	_
@@ -325637,7 +325637,7 @@
 # text = Inicialmente incolor, ao adicionar uma certa quantidade de água, este sistema pode turvar parcialmente ou totalmente, caracterizando o fim da região de microemulsão..
 # sent_id = 240-20140220-MONOGRAFIA_0-320
 1	Inicialmente	inicialmente	ADV	_	_	2	advmod	O	_
-2	incolor	incolor	ADJ	_	Gender=Masc|Number=Sing	15	advcl	O	_
+2	incolor	incolor	ADJ	_	Gender=Masc|Number=Sing	14	amod	O	_
 3	,	,	PUNCT	_	_	6	punct	O	_
 4-5	ao	_	_	_	_	_	_	_	_
 4	a	a	SCONJ	_	_	6	mark	O	_
@@ -334541,7 +334541,7 @@
 1	2.2	2.2	NUM	_	NumType=Card	2	nummod	O	_
 2	Área	Área	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 3	de	de	ADP	_	_	4	case	O	_
-4	estudo	estudo	NOUN	_	Gender=Masc|Number=Sing	2	compound	O	_
+4	estudo	estudo	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 
 # text = A localização geométrica da bacia Potiguar – Ceará e a grande área onde está localizado o estúario Barra Grande, está representado nas figuras a seguir a fim de demostrar a distancia aproximada dos campos de petróleo em águas rasas com o município de Icapuí..
 # sent_id = 411-20150702-MONOGRAFIA_0-96
@@ -351110,7 +351110,7 @@
 4	de	de	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
 6	sabão	sabão	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
-7	base	base	NOUN	_	Gender=Fem|Number=Sing	6	compound	O	_
+7	base	base	NOUN	_	Gender=Fem|Number=Sing	6	nmod	O	_
 8	(	(	PUNCT	_	_	9	punct	O	_
 9	SB	SB	PROPN	_	Gender=Fem|Number=Sing	6	appos	O	_
 10	)	)	PUNCT	_	_	9	punct	O	_
@@ -351457,7 +351457,7 @@
 7	padrão	padrão	NOUN	_	Gender=Masc|Number=Sing	6	nmod	O	_
 8	de	de	ADP	_	_	9	case	O	_
 9	sabão	sabão	NOUN	_	Gender=Masc|Number=Sing	6	nmod	O	_
-10	base	base	NOUN	_	Gender=Fem|Number=Sing	9	compound	O	_
+10	base	base	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
 11	,	,	PUNCT	_	_	3	punct	O	_
 12	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	13	aux:pass	O	_
 13	pesado	pesar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
